vdev/src/testing/integration.rs

```diff
@@ -64,7 +64,23 @@ impl IntegrationTest {
         for (key, value) in config_env(&self.env_config) {
             env_vars.insert(key, Some(value));
         }
-        let mut args = self.config.args.clone();
+
+        let mut args = self.config.args.clone().unwrap_or_default();
+
+        args.push("--features".to_string());
```

Comment:
AFAICR `args.push` supports pushing static strings too, so I think this (and below) could just be:
```suggestion
        args.push("--features");
```
IOW I don't think the `.to_string()` conversions are strictly necessary for things that are already strings.

Reply:
I don't think it's possible as I tried this and was met with the following message from the compiler:

error[E0308]: mismatched types
    --> vdev/src/testing/integration.rs:70:19
     |
     |         args.push("--features");
     |              ---- ^^^^^^^^^^^^- help: try using a conversion method: `.to_string()`
     |              |    |
     |              |    expected struct `std::string::String`, found `&str`
     |              arguments to this function are incorrect

Reply:
Oh, *sigh* I was thinking this was `std::process::Command` which can add either kind of arg, but what you have here is `Vec<String>.push` Of course you are right.